ci-operator/step-registry/agent/e2e/ha/ipv4v6/agent-e2e-ha-ipv4v6-workflow.metadata.json

```
@@ -0,0 +1,12 @@
+{
+	"path": "agent/e2e/ha/ipv4v6/agent-e2e-ha-ipv4v6-workflow.yaml",
```

```suggestion
	"path": "agent/e2e/ha/ipv4v6/agent-e2e-ha-dualstack-workflow.yaml",
```

(to be similar to other existing jobs)